lib/src/sdk_version.dart

```diff
@@ -0,0 +1,44 @@
+/// The current SDK version (private)
+const String _sdkVersion = '1.0.0';
```

Comment:
this has to stay in sync w/ `pubspec.yml` in dart; if we change to a flutter sdk it can be aware of the version programmatically